code/modules/mod/modules/modules_supply.dm

```diff
@@ -51,7 +51,7 @@
 		return
 	if(!mod.wearer.Adjacent(target))
 		return
-	if(istype(target, /obj/structure/closet/crate) || istype(target, /obj/item/delivery/big))
+	if (isstructure(target))
```

Comment:
you did this in the modsuits 2021 by @Fikou hydraulic clamp, and not the ripley one

Reply:
whoops